packages/types/src/session.ts

```diff
@@ -0,0 +1,37 @@
+import { IInterview, ILesson, IUser } from "@/index";
+
+export type Row = ILesson.Row | IInterview.Row;
+
+export type Self = ILesson.Self | IInterview.Self;
+
+export type Type = "lesson" | "interview";
+export type Id = `${Type}:${string}`;
+
+export type MemberRow = {
+  user_id: number;
+  session_id: Id;
+};
+
+export type Member = {
+  userId: number;
+  sessionId: Id;
+};
+
+export type PopuldatedMemberRow = {
+  session_id: Id;
+  user_id: number;
+  name: IUser.Row["name"];
+  image: IUser.Row["image"];
+  role: IUser.Role;
+};
+
+export type PopuldatedMember = {
+  sessionId: Id;
+  userId: number;
+  name: IUser.Self["name"];
```

Comment:
Same here. There is no need for these stuff.

Reply:
Totally agree. However, the `Type`, and `Id` declarations I still find them pretty useful. They bestow the code with some flexibility that can be beneficial for future modification. Explore the code base you will find it's being used in a plenty of spots.

Reply:
@neuodev